qemu/tests/cfg/balloon_hotplug.cfg

```diff
@@ -14,3 +16,19 @@
     test_tags = "evict enlarge"
     balloon_type_evict = evict
     balloon_type_enlarge = enlarge
```

Comment:
Seems the three parameters above are not used. Could you have a check by the way?

Reply:
These parameters are used for balloon_check subtest.